tasks/files/snapshot.py

```diff
@@ -2076,6 +1855,107 @@ def validate_snapset_json(cmd, snapset, verify_only):
     return rc, message, snapset_json
 
 
+def get_mount_json_from_args(args):
+    volume_list = []
+    args_json = {}
+    volume = {}
+
+    if args.suffix:
+        args_json["name"] = args.suffix
+
+    if hasattr(args, "create"):
```

Comment:
if you know that `args` is from the `mount` subcommand and used the `mount_parser` argument parser, then `hasattr(args, "field")` will return `True` for all of the fields - they will all be present in `args` with either the given value, or the default value, or `None` if no default value was specified with `add_argument`.

Reply:
ok - thanks